flake.nix

```diff
@@ -50,6 +50,7 @@
             packages = p: with p; [ hspec-discover ];
             buildInputs = [
               cabal-install
+              ccls
```

Comment:
This doesn't belong in a GH action pull request. Also I wouldn't use ccls, but clangd instead.

Reply:
removing ...